Doc/library/typing.rst

```diff
@@ -2114,6 +2114,31 @@ Functions and decorators
    runtime we intentionally don't check anything (we want this
    to be as fast as possible).
 
+.. function:: assert_type(val, typ, /)
+
+   Assert (to the type checker) that *value* has an inferred type of *type*.
```

Comment:
```suggestion
   Assert (to the type checker) that *value* has an inferred type of *typ*.
```
The parameter name is `typ`, not `type`

Reply:
Good catch! Also, the other param is named `val`, not `value`. I chose those names because `cast()` uses them too.
```suggestion
   Assert (to the type checker) that *val* has an inferred type of *typ*.
```